app/commands/todo/add.rb

```diff
@@ -18,7 +22,8 @@ def initialize(params)
             @aggregate_id = payload.delete(:todo_id)
           end
 
-          def valid?
+          def validate
+            raise BadRequest, 'todo_id is blank' if aggregate_id.nil?
```

Comment:
Can we add some specs for these? I don't know if it will ever get here as it's part of the path.